Sources/AsyncHTTPClient/ConnectionPool/HTTPConnectionPool.swift

```diff
@@ -130,14 +130,7 @@ final class HTTPConnectionPool {
     }
 
     private let stateLock = Lock()
-    private var _state: StateMachine {
-        didSet {
-            self.logger.trace("Connection Pool State changed", metadata: [
-                "key": "\(self.key)",
-                "state": "\(self._state)",
-            ])
-        }
```

Comment:
Removed, since it produces way too many logs. We should solely focus on recording the events so that we can replay the finite state machine if needed.